docs/plugins/main.py

```diff
@@ -113,29 +113,31 @@ def install_logfire(markdown: str, page: Page) -> str:
         package = 'logfire' if not extras else f"'logfire[{','.join(extras)}]'"
         extras_arg = ' '.join(f'-E {extra}' for extra in extras)
         instructions = f"""
-=== "PIP"
+=== "pip"
     ```bash
     pip install {package}
     ```
 
-=== "Rye"
+=== "uv"
     ```bash
-    rye add logfire {extras_arg}
+    uv add {package}
     ```
 
-=== "Poetry"
+=== "rye"
     ```bash
-    poetry add {package}
+    rye add logfire {extras_arg}
     ```
 
-=== "uv"
+=== "poetry"
     ```bash
-    uv add {package}
+    poetry add {package}
     ```
+
 """
         if not extras:
             instructions += """
-=== "Conda"
+
+=== "conda"
     ```bash
     conda install -c conda-forge logfire
     ```
```

Comment:
☝️ Moving these above the conditional line fixed the python linter error @Kludex. Rendered bash commands looks correct to me now, would appreciate a double-check